doc/mac-build.md

```diff
@@ -5,7 +5,7 @@
 
    ```
    brew install pkg-config
-   brew install libsodium
+   brew install https://raw.githubusercontent.com/Homebrew/homebrew-core/65effd2b617bade68a8a2c5b39e1c3089cc0e945/Formula/libsodium.rb   
```

Comment:
Please add comment about reason to set libsodium from url. (non-last 1.0.12 version required)